rag/utils/redis_conn.py

```
@@ -473,29 +490,96 @@ def get_unacked_iterator(self, queue_names: list[str], group_name, consumer_name
+                    last_id = last_id.decode()
+                else:
+                    last_id = str(last_id)
+                if last_id == previous_last_id:
+                    break
+                previous_last_id = last_id
+                current_min = f"({last_id}"
+        except redis.exceptions.ResponseError as e:
+            error = str(e).lower()
+            if "no such key" not in error and "nogroup" not in error:
+                logging.warning("RedisDB.get_pending_msg " + str(queue) + " got exception: " + str(e))
+        except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)